src/shared/utilities/pathUtils.ts

```diff
@@ -24,6 +36,27 @@ export function dirnameWithTrailingSlash(path: string): string {
     return dirname
 }
 
+/**
+ * Normalizes path `p`:
+ * - Replaces backslashes "\\" with "/".
+ * - Removes redundant path separators (except initial double-slash for UNC-style paths).
+ * - Uppercases drive-letter (Windows).
```

Comment:
Have you tried this on Windows? I think we need lowercase drive paths (at the very least, capital drive paths in the launch configs have been breaking for me). I can do some investigation if necessary.

Reply:
either way is fine, but perhaps we should confirm this. Some code explicitly states that uppercase is needed. 

created a tracking issue.

Reply:
Looks like this is due to how the cftRegistry is handling paths (as in: always lowercasing). Will pipe that and all getRegistry calls through the path.normalize function.